sormas-backend/src/main/java/de/symeda/sormas/backend/contact/ContactFacadeEjb.java

```diff
@@ -1088,4 +1090,9 @@ public void validate(ContactDto contact) throws ValidationRuntimeException {
 
     }
 
+	@Override
+	public Boolean isContactEditAllowed(String contactUuid) {		
```

Comment:
Use boolean instead. I don't this there is a situation where returning null would help.

Reply:
changed